inspirehep/modules/workflows/workflows/article.py

```diff
@@ -310,20 +316,20 @@
     ),
 ]
 
-CHECK_IF_MERGE_AND_STOP_IF_SO = [
+CHECK_IF_MERGE = [
     IF(
         article_exists,
         [
             IF_ELSE(
                 is_submission,
                 NOTIFY_ALREADY_EXISTING,
                 [
-                    # halt_record(action="merge_approval"),
-                    delete_self_and_stop_processing,
+                    merge_articles
                 ]
             ),
-        ]
-    )
+        ],
+    ),
+    store_temporary_root
```

Comment:
Put before and removed the assignments in the `merge_article` function, because not needed anymore. :+1: